gmso/core/element.py

```diff
@@ -26,10 +26,11 @@ class Element(GMSOBase):
     mass: u.unyt_quantity = Field(..., description='Mass of the element.')
 
     def __repr__(self):
+        mass_string = f"{self.mass.to(u.amu)}".split()[0] + " amu"
```

Comment:
I think we can use the unyt api rather than doing string splits. Something like
```py
mass_amu = self.mass.to(u.amu)
mass_string = f"{mass_amu.value} {mass_amu.units}" 
```

Reply:
Actually that does not work in this case. It seems to treat `amu` and `elementary charge` as physical constants, so it still makes the string ugly. Try it out and you'll see. Thats why I needed the hacky workaround to begin with.